nav2_system_tests/src/localization/test_localization_launch.py

```diff
@@ -17,56 +17,81 @@
 import os
 import sys
 
+from ament_index_python.packages import get_package_share_directory
+
 from launch import LaunchDescription
 from launch import LaunchService
-import launch.actions
-from launch.actions import ExecuteProcess
-import launch_ros.actions
+from launch.actions import ExecuteProcess, SetEnvironmentVariable
+from launch_ros.actions import Node
 from launch_testing.legacy import LaunchTestService
 
 
+def generate_launch_description():
+    aws_dir = get_package_share_directory('aws_robomaker_small_warehouse_world')
+    map_yaml_file = os.path.join(aws_dir, 'maps', '005', 'map.yaml')
+    world = os.path.join(aws_dir, 'worlds', 'small_warehouse', 'small_warehouse.world')
+
+    urdf = os.getenv('TEST_URDF')
+    with open(urdf, 'r') as infp:
+        robot_description = infp.read()
+
+    return LaunchDescription([
+        SetEnvironmentVariable('RCUTILS_LOGGING_BUFFERED_STREAM', '1'),
+        SetEnvironmentVariable('RCUTILS_LOGGING_USE_STDOUT', '1'),
+
+        # Launch gazebo server for simulation
+        ExecuteProcess(
+            cmd=['gzserver', '-s', 'libgazebo_ros_init.so', '-s', 'libgazebo_ros_factory.so',
+                 world],
```

Comment:
You removed minimal comms from the gazebo call, that's really important for CI since we have limited compute

Reply:
Also in all other files you removed it too, please add back

Reply:
Added `--minimal_comms` back.